src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+      {person.isMarried ? (
+        <>
+          <p className="Person__partner">
+            {person.partnerName} is my {person.sex === 'm' ? 'wife' : 'husband'}
```

Comment:
nester ternary looks complicated, let's create a separate variable and use it here

`const partner = person.sex === 'm' ? 'wife' : 'husband'`